x/ibc/07-tendermint/types/header.go

```diff
@@ -15,23 +15,31 @@ import (
 var _ clientexported.Header = Header{}
 
 // Header defines the Tendermint consensus Header
+// Header encapsulates all the information necessary to update from a trusted Tendermint ConsensusState
+// The SignedHeader and ValidatorSet are the new untrusted update fields for the client
+// The TrustedHeight is the height of a stored ConsensusState on the client that will be used to verify new untrusted header
+// The Trusted ConsensusState must be within the unbonding period of current time in order to correctly verify,
+// and the TrustedValidators must hash to TrustedConsensusState.NextValidatorsHash since that is the last trusted validatorset
+// at the TrustedHeight
```

Comment:
```suggestion
// Header defines the Tendermint client consensus Header.
// It encapsulates all the information necessary to update from a trusted Tendermint ConsensusState.
// The SignedHeader and ValidatorSet are the new untrusted update fields for the client.
// The TrustedHeight is the height of a stored ConsensusState on the client that will be used to verify the new untrusted header.
// The Trusted ConsensusState must be within the unbonding period of current time in order to correctly verify,
// and the TrustedValidators must hash to TrustedConsensusState.NextValidatorsHash since that is the last trusted validator set
// at the TrustedHeight.
```